docs/zero_to_hero_guide/quickstart.md

```diff
@@ -22,15 +22,23 @@ If you're looking for more specific topics like tool calling or agent setup, we
    - Download and unzip `Ollama-darwin.zip`.
    - Run the `Ollama` application.
 
-2. **Download the Ollama CLI**:
+1. **Download the Ollama CLI**:
    - Ensure you have the `ollama` command line tool by downloading and installing it from the same website.
 
-3. **Verify Installation**:
+1. **Start ollama server**:
+   - Open the terminal and run:
+      ```
+      ollama serve
+      ```
+
+1. **Verify Installation**:
    - Open the terminal and run:
      ```bash
-     ollama run llama3.2:1b
+     ollama run llama3.2:3b-instruct-fp16
      ```
-
+     Note:
+     1. The supported models for llama stack for now is listed in [here](https://github.com/meta-llama/llama-stack/blob/acbecbf8b3217d2594bad3eee8a322e16b8ee725/llama_stack/providers/remote/inference/ollama/ollama.py#L42-L69)
```

Comment:
this link looks odd since it is pointing to a specific commit vs. on main/ --- can you update it?

Reply:
Updated. Thanks for the heads up